database/migrations/2020_10_01_180409_copy_degrees_and_courses_to_experiences_education.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CopyDegreesAndCoursesToExperiencesEducation extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $degrees = DB::table('degrees')->get();
+
+        foreach ($degrees as $degree) {
+
+            if ($degree->blockcert_url) {
+                $blockcert = true;
+            } else {
+                $blockcert = false;
+            }
+
+            DB::table('experiences_education')->insert([
+                'education_type_id' => $degree->degree_type_id,
+                'area_of_study' => $degree->area_of_study,
+                'institution' => $degree->institution,
+                'education_status_id' => 1,
```

Comment:
I'd prefer to get the id with a query, something like `$complete_id = DB::table('education_statuses)->where('name', 'complete_credited')->first()->id` (that's probably the wrong syntax). Just makes it a bit more robust and readable.

Reply:
You got it.